sdk/highlight-go/go.mod

```diff
@@ -6,6 +6,8 @@ require (
 	github.com/99designs/gqlgen v0.17.24
 	github.com/gin-gonic/gin v1.7.7
 	github.com/gofiber/fiber/v2 v2.42.0
+	github.com/highlight-run/highlight/backend v0.0.0-20230707192428-b66e47a33c89
```

Comment:
we don't want the SDK to depend on the backend code. this might have been automatically added if you didn't set up go workspaces?

Reply:
I see that the main branch now has this dependency. As such, I would like to resolve this conversation point.